content/doc/book/pipeline/development.adoc

```diff
@@ -0,0 +1,145 @@
+---
+layout: section
+title: Pipeline Development Tools
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Pipeline Development Tools
+
+In this section, we'll discuss tools and resources
+that help speed development of Jenkins Pipelines.
+
+== Blue Ocean Editor (Declarative only)
+
+The
+<<../blueocean/pipeline-editor#, Blue Ocean Pipeline Editor>> provides a WYSIWYG
+way to create Declarative Pipelines. It gives you a structural view of all the stages,
+parallel branches, and steps in your Pipeline. It validates Pipeline changes as they are
+made, eliminating many errors before they are even committed.  Behind the scenes
+it still generates Declarative Pipeline code.
+
+[[linter]]
+== Command-line Pipeline Linter (Declarative only)
+
+Jenkins can validate, or
+"link:https://en.wikipedia.org/wiki/Lint_(software)[lint]",
+a Declarative Pipleline from the command line before running actually running it.
+This can be done using a Jenkins CLI command or using `curl` to post the Jenkinsfile directly.
```

Comment:
Technically one doesn't need to use `curl` but rather make an HTTP POST request with the appropriate parameters. I think the wording should reflect that